ydb/docs/en/core/reference/docker/configuration.md

```diff
@@ -18,6 +18,7 @@
 | [`YDB_ERASURE`](https://GitHub.com/ydb-platform/ydb/blob/8fefc809c83829d8d8b886e82534d009de4c8826/ydb/public/tools/lib/cmds/__init__.py#L50) | `string` | `none` | Erasure to use, see [{#T}](../../concepts/topology.md) |
 | [`FQ_CONNECTOR_ENDPOINT`](https://GitHub.com/ydb-platform/ydb/blob/c113fcffa7b1a20ad8dcb1b1760ae5bfa25370ca/ydb/public/tools/lib/cmds/__init__.py#L261) | `string` | `None` | Connection string for the connector to external sources `fq-connector-go`, see [{#T}](../../concepts/federated_query/index.md) |
 | [`YDB_USE_IN_MEMORY_PDISKS`](https://GitHub.com/ydb-platform/ydb/blob/c113fcffa7b1a20ad8dcb1b1760ae5bfa25370ca/ydb/public/tools/lib/cmds/__init__.py#L230) | `0` or `1` | `0` | Makes all data volatile and stored only in RAM. Currently, saving data by disabling this option is supported only on x86_64 processors or virtual machines emulating them. |
+| [`YDB_PDISK_SIZE`](https://github.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L23-L27) | `string` | `64GB` | The size of the file for storing data in the `/ydb_data` directory used by the running container. It can be specified as a number in bytes or with a suffix: `KB`, `MB`, `GB`, or `TB` (for example, `64GB`). |
```

Comment:
```suggestion
| [`YDB_PDISK_SIZE`](https://github.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L23-L27) | `string` | `64GB` | The size of the file for storing data in the `/ydb_data` directory used by the running container. It can be specified as a number in bytes or with a suffix: `KB`, `MB`, or `GB` (for example, `64GB`). |
```